content/swiftui/concepts/views/terms/button/button.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: 'Button'
+Description: 'A very basic component in UI, which is used to interact and perform certain actions'
+Subjects:
+  - 'Sotfware Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Buttons'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A very basic component in UI, which is used to interact and perform certain actions, such as logging into a page, or deleting a file etc.
+
+## Syntax
+
+```swift
+Button("Text inside the button"){
+  //What happens when you click the button
+}
+```
+
+The `Text inside the button` is what is shown inside the button.
+
+## Example
+
+To make a `'delete'` button:
+
+```swift
+    Button("Delete") {
+     //Deletes a file
+}
+ .font(.title)
+ .padding()
+ .backgroundColor(Color.red)
+ .cornerRadius(40)
+ .foregroundColor(Color.white)
+ .overlay(RoundedRectangle(cornerRadius: 40).stroke(Color.red))
+```
+
+## Codebyte Example
+To `count` the number of times we `click` the button:
+
+
+```codebyte/swiftui
```

Comment:
Codebytes do not support SwiftUI at this time. (The list of supported languages is listed in [this document.](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md))

I'd suggest using this as the main example, with an extra sentence describing what it does.

Reply:
@girijakar you still need to address this. There's no "codebyte/swiftui" avalible. This needs to be changed to a regular example. As I suggest before, I would replace the prior example with this one. You cannot have a swiftui codebyte example.

Reply:
Ohk,sorry for that. Just clarifying, I need to keep the example code in single quote right?